ols/app/models/config.py

```diff
@@ -366,8 +366,8 @@ def __init__(self, data: Optional[dict] = None) -> None:
             return
         self.host = data.get("host", constants.REDIS_CACHE_HOST)
 
+        yaml_port = data.get("port", constants.REDIS_CACHE_PORT)
```

Comment:
i don't understand why this is necessary?

it seems to work fine here as is:

https://www.online-python.com/Y1PO2MxmAC

i could see an issue if the variable was used outside the try block, but it is not.

Reply:
it works only because the exception is thrown after variable initialization. But if new code will be added in the try block, it won't work.